Dockerfile

```
@@ -0,0 +1,16 @@
+FROM gradle:6.6-jdk11 AS build
```

```suggestion
FROM gradle:6.8-jdk11 AS build
```
Please see th2-box-template

---

> Please see th2-box-template
You mean that some modifications to `build.gradle` are required in order to migrate to Gradle 6.8?

---

Modifications are optional but nice to have.